src/components/checkbox/checkbox.tsx

```diff
@@ -226,7 +226,7 @@ export class Checkbox implements LabelableComponent, CheckableFormComponent, Int
           role="checkbox"
           tabIndex={this.disabled ? undefined : 0}
         >
-          <svg class="check-svg" viewBox="0 0 16 16">
+          <svg aria-hidden="true" class="check-svg" role="none" viewBox="0 0 16 16">
```

Comment:
We won't need the `role` attribute for Chrome/VoiceOver skip over the svg. So if we're looking to limit attributes, we can get this one down to one new attribute. ✂️ 

`<svg aria-hidden="true" class="check-svg" viewBox="0 0 16 16">`